tests/cli/test_install_notebooks.py

```diff
@@ -6,13 +6,15 @@
 
 def test_install_notebooks():
     with TemporaryDirectory() as tmpdir:
-        expected_filename = "registration.ipynb"
+        expected_filenames = ["registration.ipynb", "start_notebook.sh"]
         check_run_command(f"dsgrid install-notebooks --path={tmpdir}")
         files = list((Path(tmpdir) / "dsgrid-notebooks").iterdir())
-        assert files
-        assert files[0].name == expected_filename
+        # TODO: Why does this fail only on CI?
```

Comment:
This passes on my local system as well as on Eagle. If you have an idea about what is happening, let me know.